sound/soc/sof/intel/hda.c

```diff
@@ -179,6 +179,40 @@ void hda_dsp_dump(struct snd_sof_dev *sdev, u32 flags)
 	}
 }
 
+void hda_ipc_regs_dump(struct snd_sof_dev *sdev)
+{
+	u32 hipci;
+	u32 hipcie;
+	u32 hipct;
+	u32 hipcte;
+	u32 hipcctl;
+
+	/* read IPC status */
+	hipcie = snd_sof_dsp_read(sdev, HDA_DSP_BAR,
+				  HDA_DSP_REG_HIPCIE);
+	hipct = snd_sof_dsp_read(sdev, HDA_DSP_BAR, HDA_DSP_REG_HIPCT);
+	hipcctl = snd_sof_dsp_read(sdev, HDA_DSP_BAR, HDA_DSP_REG_HIPCCTL);
+
+	hipci = snd_sof_dsp_read(sdev, HDA_DSP_BAR,
+				 HDA_DSP_REG_HIPCI);
+
+	hipcte = snd_sof_dsp_read(sdev, HDA_DSP_BAR,
+				  HDA_DSP_REG_HIPCTE);
```

Comment:
parameter alignment seems inconsistent?

Reply:
not sure if we really need them. Since not all of them are useful